### ISSUE TITLE
does loading cmip5 data actually work?

### ISSUE THREAD

**Issue by mathause** [MESMER-group/mesmer#89]
In `find_files_cmipng` for cmip5 you only glob `path_runs_list`

https://github.com/MESMER-group/mesmer/blob/04240a4633292f2ed5058a72e3f5292e48ee1e62/mesmer/io/load_cmipng.py#L101

but further down you loop `path_runs_list_scen` which does not exist for cmip5:

https://github.com/MESMER-group/mesmer/blob/04240a4633292f2ed5058a72e3f5292e48ee1e62/mesmer/io/load_cmipng.py#L190

so I think this code path is dead. Do we need to salvage this? @leabeusch

**Comment by leabeusch** [comment]
Yes, it is unfortunately dead and yes, we definitely need to salvage it. I still have a paper in review with CMIP5 data and I wanted to retrain MESMER for the revisions with the most up-to-date code version -> I'll try to draft a PR to make it work again in the next few days, ok?

**Comment by leabeusch** [comment]
Ok, had a closer look at this now & as stated, reading in cmip5 data was not actually working anymore. However, I don't think there is any issue with `path_runs_list`?! The loop over `path_runs_list_scen` is done as part of the `if gen == 6:` statement and thus is only carried out for cmip6, where `path_runs_list_scen` does exist. However, there were several other issues (thanks for your contribution to solving them btw!), which should all be addressed by PR #90. Hopefully my fixes don’t interfere with your work on PR #55 in an annoying way.
